### ISSUE TITLE
Invert text color on tiles with clashing background colors

### ISSUE THREAD

**Issue by MusikAnimal** [MusikAnimal/pageviews#25]
For darker colors, you can't see the text on the Select2 and chart legend tiles. We need to come up with some smart logic to figure out when to invert the text color. We could even consider some fancy pancy CSS3 filters... have to play around and see if it can do what I think it can do.

Other solutions would be to hard-code which colors require the text color to be inverted. This far from ideal, however, as it sort of complicates the way we have our config set up. Perhaps an additional array of text colors to go with the arrays of background colors.

**Comment by kaldari** [comment]
This will be pretty tricky to implement as we'll actually have to come up with 3 different inversions (for link color, text color, and x color) all of which are distinct from the 10 background colors, and figure out some crazy logic to switch them. The 3rd color scheme that you came up with almost doesn't need any inversions, except for the blue color which is too similar to the link color. I played around with the colors some more and came up with the following palette, which doesn't need any inversions and has all distinct colors:
`'1': ['rgba(171, 212, 235, 1)', 'rgba(178, 223, 138, 1)', 'rgba(251, 154, 153, 1)', 'rgba(253, 191, 111, 1)', 'rgba(202, 178, 214, 1)', 'rgba(207, 182, 128, 1)', 'rgba(141, 211, 199, 1)', 'rgba(252, 205, 229, 1)', 'rgba(255, 248, 164, 1)', 'rgba(217, 217, 217, 1)'],`
Basically: light blue, light green, light red, orange, light purple, light brown, aquamarine, pink, yellow, light gray.
Perhaps we could just standardize the colors as something similar to that and remove the customization option.

**Comment by kaldari** [comment]
Here is a screenshot:
![screen shot 2016-02-21 at 12 06 08 am](https://cloud.githubusercontent.com/assets/1707155/13201270/077afd28-d82f-11e5-9378-0ee634d02954.png)

**Comment by MusikAnimal** [comment]
Nice! Removing the color options would certainly clean up the interface, and this palette seems to be a happy medium for everyone. I will get this out soon! Thank you

**Comment by kaldari** [comment]
I went ahead and implemented this in https://github.com/MusikAnimal/pageviews/commit/473e1622cf6f122eb42c8b0dd58d8efd0e77e87d (hope that's OK).

**Comment by MusikAnimal** [comment]
No problem, I had been meaning to take care of this myself. I've deployed it to prod. We should also update the image tiles in "Change chart type", but first let's see if anyone complains about the new colours. I doubt they will, but ya never know
